core-js/src/main/javascript/cdf/components/UnmanagedComponent.js

```diff
@@ -574,8 +574,14 @@ define([
       query.setAjaxOptions(ajaxOptions);
 
       // Other Query Options
-      if(queryOptions && queryOptions.pageSize)
-        query.setPageSize(queryOptions.pageSize);
+      if(queryOptions) {
+        if(queryOptions.pageSize) {
+          query.setPageSize(queryOptions.pageSize);
+        }
+        if(queryOptions.searchPattern) {
```

Comment:
@jvelasques does it makes sense to have the search param here? it is only used in a component and probably doesn't make sense to have it here